fetch.bs

```diff
@@ -5223,6 +5688,10 @@ constructor must run these steps:
  <code>credentials</code> member if it is present, and
  <var>fallbackCredentials</var> otherwise.
 
+ <li><p>If <var>useReferredTokenBinding</var> is true, set <var>request</var>'s
+ <a for=request>referred-token-binding origin</a> to
+ <var>origin</var>.
+
```

Comment:
Nit: move this below the next list item so that the credentials bits stay together.

Reply:
Done.